cmd/aws-iam-authenticator/token.go

```diff
@@ -36,13 +36,19 @@ var tokenCmd = &cobra.Command{
 		clusterID := viper.GetString("clusterID")
 		tokenOnly := viper.GetBool("tokenOnly")
 		forwardSessionName := viper.GetBool("forwardSessionName")
+		sessionName := viper.GetString("sessionName")
 		cache := viper.GetBool("cache")
 
 		if clusterID == "" {
 			fmt.Fprintf(os.Stderr, "Error: cluster ID not specified\n")
 			cmd.Usage()
 			os.Exit(1)
 		}
+		if forwardSessionName && sessionName != "" {
```

Comment:
Please leave one blank line between the if blocks. Also please fix the indentation for cmd.Usage and os.Exit calls.